draft/elm.md

```diff
@@ -0,0 +1,30 @@
+# Elm, functional frontend development
+
+## … and why you should care
+
+A few months ago I got to know about [Elm](http://elm-lang.org/), which claims to be "the best of functional programming in your browser". After inhaling the documentation and watching a few videos and talks, I started tinkering myself and quickly fell in love with [Elm, functional frontend development](https://dennisreimann.de/articles/elm.html), and the feeling it gives me when writing code in it.
```

Comment:
Suggest:

> A few months ago <strike>I got to know about</strike> I learned of Elm 

Also:

> quickly fell in love with [Elm, functional frontend development](https://dennisreimann.de/articles/elm.html), and the feeling it gives me when writing code in it.

This reads a bit odd. I would suggest keeping the link to your site, but adjust to remove the full title. Something like this:

> quickly [fell in love](https://dennisreimann.de/articles/elm.html) with Elm and the feeling it gives me when writing code in it.